### ISSUE TITLE
Provide option for different units in scale bar

### ISSUE THREAD

**Issue by beyhh** [qgis/QGIS#37064]
It would be nice if the scale bar (View > Decorations > Scale Bar...) would allow units other than _m_/_km_. 
In several applications it might be useful to have a scale bar in _Nautical Miles_ and _Cables_ for instance or American and British units.

**Comment by gioman** [comment]
@beyhh 

![Screenshot_20200609_111553](https://user-images.githubusercontent.com/1951107/84136022-a1f56980-aa42-11ea-8c3e-69a823764060.png)

**Comment by gioman** [comment]
ah sorry, you were speaking about decorations.

**Comment by beyhh** [comment]
Yes. The scale bar wich is available via Decorations and which is basically a "floating" scale bar on the map canvas.
